pkg/cmd/objects/browse/browse.go

```diff
@@ -43,8 +43,17 @@ func NewBrowseCmd(f *cmdutil.Factory) *cobra.Command {
 			This command browse the objects of the specified index.
 		`),
 		Example: heredoc.Doc(`
```

Comment:
Can we add an example for 'exporting the objects to a file':

`algolia objects browse BOOKS > books.ndjson`

Reply:
Sure! 🚀